source/spp/mnm/spp_116_nl_abiotiek_stilstaand/03_benodigdheden.Rmd

```diff
@@ -0,0 +1,58 @@
+# Extra benodigdheden t.o.v. de subprotocols
+
+Tabel \@ref(tab:Tabel1) geeft een overzicht van de benodigde apparatuur en materiaal voor het uitvoeren van een abiotische staalname.
+
++:-----------------------------------------------------+
+| **Algemene benodigdheden**                           |
++------------------------------------------------------+
+| -   Staalnameformulier                               |
++------------------------------------------------------+
+| -   Klembord, potlood en slijper                     |
++------------------------------------------------------+
+| -   Laarzen, lieslaarzen, waadpak (site-afhankelijk) |
++------------------------------------------------------+
+| -   Latex handschoenen (poederloos)                  |
++------------------------------------------------------+
+| -   Plooimeter                                       |
++------------------------------------------------------+
+| -   Fototoestel                                      |
++------------------------------------------------------+
+| -   Documentatie lokalisatie staalnamepunt           |
++------------------------------------------------------+
+
+: (#tab:Tabel1) Checklist benodigd veldmateriaal.
+
+## Locatieafhankelijke benodigdheden
+
+Afhankelijk van de diepte en aard van de staalnamelocatie en de ligging van het staalnamepunt kunnen laarzen, lieslaarzen, waadpak en in bepaalde gevallen zelfs een boot nodig zijn.
+Bij werk vanop een boot of bij steile oevers kan een zwemvest en/of reddingstouw aangeraden of verplicht zijn (zie protocol Veiligheid in en rond water [SFP-112]()).
```

Comment:
was er hiervan al een versie waar je naar kan verwijzen?

Reply:
nee, dat protocol is nog niet opgemaakt

Reply:
Misschien kan je hier dan gewoon schrijven dat het protocol nog in opmaak is?

Reply:
Ja, ok. Ik had de referenties naar de protocollen weggelaten indien ze nog niet beschikbaar waren, maar ik zal het er zo bij zetten

Reply:
hiernaar is ook verwezen bij sfp-114 en sfp-115; je kan dit wellicht op dezelfde wijze doen